CHANGELOG.txt

```diff
@@ -1,5 +1,20 @@
-UNRELEASED
-----------
+3.6.0 (UNRELEASED)
+------------------
+
+* Two new functions are added to the ``asgiref.sync`` module: ``iscoroutinefunction()``
+  and ``markcoroutinefunction()``.
+
+  Python 3.12 deprecates asyncio.iscoroutinefunction() as an alias for
+  inspect.iscoroutinefunction(), whilst also removing the _is_coroutine marker.
```

Comment:
```suggestion
  ``inspect.iscoroutinefunction()``, whilst also removing the ``_is_coroutine`` marker.
```

Should these two be in backticks?

Similar below too.


Side comment.  Whilst. 😅👍